qa-tests-backend/src/test/groovy/DefaultPoliciesTest.groovy

```diff
@@ -128,6 +137,16 @@ class DefaultPoliciesTest extends BaseSpecification {
         gcrId = GCRImageIntegration.createDefaultIntegration()
         assert gcrId != ""
 
+        ImageService.clearImageCaches()
```

Comment:
Is it possible to do this just for the test that failed instead of all of them. This is a pretty hefty test file and this deletion can slow it down quite a bit.

Reply:
I think setupSpec() runs just for the test file, so it should only run once. Do you mean remove and add back just the struts image which causes the specific failure instead of all images ?

Reply:
Oh I missed the fact that it was setupSpec